tools/rna_tools/targetfinder/TargetFinder_threads.xml

```diff
@@ -0,0 +1,77 @@
+<tool id="targetfinder_threads" name="Plant small RNA target prediction tool" version="0.1.0">
```

Comment:
Since you only have now one tool, I would call it `targetfinder` and use `targetfinder_threads.pl`. No one will care about this implementation detail, but will wonder why this tool has such a strange ID or name (here I mean file name) :)